src/rviz_visual_tools_demo.cpp

```diff
@@ -358,7 +358,33 @@ class RvizVisualToolsDemo : public rclcpp::Node
     */
 
     // --------------------------------------------------------------------
-    // TODO(davetcoleman): publishMesh
+    RCLCPP_INFO(get_logger(), "Displaying Meshes");
+    pose1 = Eigen::Isometry3d::Identity();
+    y += space_between_rows;
+    pose1.translation().y() = y;
+
+    step = 0.2;
+    const std::array<double, 6> thetas = { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 };
+    const std::array<std::string, 6> meshes = {
+      "pawn", "knight", "bishop", "rook", "queen", "king"
```

Comment:
I'm not a huge fan of uploading so many STL's, I think one would have been fine. But I guess it's fine